recipes/install.rb

```diff
@@ -18,9 +18,18 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+# This way we force to have updated indexes
+include_recipe 'apt'
```

Comment:
Is this required? `apt-get update` should be run automatically when adding the repository below.